x-pack/plugins/security_solution/public/common/components/event_details/get_alert_summary_rows.tsx

```diff
@@ -271,43 +273,24 @@ export const getSummaryRows = ({
         if (field.id === 'agent.id' && !isAlertFromEndpointEvent({ data })) {
           return acc;
         }
-
-        if (field.id === `${ALERT_THRESHOLD_RESULT}.terms`) {
-          try {
-            const terms = getOr(null, 'originalValue', item);
-            const parsedValue = terms.map((term: string) => JSON.parse(term));
-            const thresholdTerms = (parsedValue ?? []).map(
-              (entry: { field: string; value: string }) => {
-                return {
-                  title: `${entry.field} [threshold]`,
-                  description: {
-                    ...description,
-                    values: [entry.value],
-                  },
-                };
-              }
-            );
-            return [...acc, ...thresholdTerms];
-          } catch (err) {
-            return [...acc];
-          }
+        if (field.id === `${ALERT_THRESHOLD_RESULT}.terms` && hasThresholdResult(rawEventData)) {
```

Comment:
Is it a requirement for this feature to be backwards compatible with alert fields prefixed with `signal` instead of `kibana.alert`?

I don't have access to legacy data in my test environment, so I can't assert whether or not equivalent (legacy) threshold alerts contained the same data, but prefixed with `signal` instead of `kibana.alert`, however IF both of the following are true:

* This feature is required to be backwards compatible 
* Legacy alerts contained the same data, but prefixed with `signal` instead of `kibana.alert`

would checks like this one and most places where the `ALERT_THRESHOLD_RESULT` is referenced throughout this PR require branching logic to access the `signal`-prefixed version of the field?